central/sensor/service/connection/upgradecontroller/trigger.go

```diff
@@ -129,11 +129,11 @@ func (u *upgradeController) doTriggerUpgrade() (common.MessageInjector, *central
 	case storage.ClusterUpgradeStatus_SENSOR_VERSION_HIGHER:
 		// We still allow upgrade triggers in this case.
 	case storage.ClusterUpgradeStatus_MANUAL_UPGRADE_REQUIRED:
-		return nil, nil, errors.Errorf("manual upgrade required for cluster %s; cannot trigger upgrade", u.clusterID)
+		return nil, nil, errors.Errorf("RHACS does not manage version for secured cluster %s; cannot trigger upgrade/downgrade", u.clusterID)
```

Comment:
```suggestion
		return nil, nil, errors.Errorf("RHACS does not control the version for the secured cluster %s; cannot trigger upgrade or downgrade", u.clusterID)
```

Reply:
Addressed in [f81eac1](https://github.com/stackrox/stackrox/pull/12833/commits/f81eac1c3bb3705d1a08dd84fbc5e019187b768f)